LICENSE.md

```diff
@@ -0,0 +1,15 @@
+Licensed under the apache-2.0 license, except as noted below.  
```

Comment:
@ejaronne do we still need the last 3 paragraphs from [this License.md](https://github.com/CMSgov/cms-ars-5.0-microsoft-windows-server-2019-stig-overlay/blob/main/LICENSE.md)?

Reply:
Done